packages/api/src/routes/middlewares/auth.ts

```diff
@@ -6,31 +6,38 @@ import * as jwt from "jsonwebtoken";
 import { MAPIAccess } from "../../models/medical/mapi-access";
 import { Config } from "../../shared/config";
 import { getCxIdOrFail } from "../util";
+import { getAuth, getCxId, PropelAuth } from "./propelauth";
 
 /**
  * Process the API key and get the customer id.
  * The customer id is stored on the Request, property 'cxId'.
  */
-export function processCxId(req: Request, res: Response, next: NextFunction): void {
+export async function processCxId(req: Request, res: Response, next: NextFunction): Promise<void> {
+  // validate it has the needed info
+  const auth = getAuth();
```

Comment:
Should we move this to where it's used? Is there a risk that propel auth going down prevents event MAPI API requests if this is here?

Reply:
Good point. I didn't want to fail silently on JWT verification, but I'll update this as I agree we shouldn't let that fail M2M requests (machine-to-machine)

Reply:
updated